fastlane-plugin-wpmreleasetoolkit.gemspec

```diff
@@ -35,7 +35,7 @@ Gem::Specification.new do |spec|
   # spec.add_dependency 'your-dependency', '~> 1.0.0'
   spec.add_dependency 'diffy', '~> 3.3'
   spec.add_dependency 'nokogiri', '>= 1.10.4'
-  spec.add_dependency 'octokit', '~> 4.13'
+  spec.add_dependency 'octokit', '~> 4.18'
```

Comment:
Need to bump this in the `.gemspec` to ensure its resolved to at least 4.18, which is the version that fixes the issue with faraday 1.0.0, which comes from Fastlane.

Reply:
To avoid this sort of issue in the future, WDYT about relaxing this dependency to just be `~> 4` which would take any compatible version in the 4.xx series that's compatible with the other gems? I don't think we require any specific functionality from `4.18` – moreso just the compatibility?